packages/SwingSet/src/vats/network/network.js

```diff
@@ -385,10 +385,16 @@ export function makeNetworkProtocol(protocolHandler) {
         const [port, listener] = listening.get(listenPrefix);
         let localAddr;
         try {
-          // See if we have a listener that's willing to receive this connection.
+          // See if our protocol is willing to receive this connection.
           // eslint-disable-next-line no-await-in-loop
-          const localSuffix = await E(listener)
-            .onInbound(port, listenPrefix, remoteAddr, listener)
+          const localSuffix = await E(protocolHandler)
```

Comment:
I find myself looking for a test again. What would it cost?

Reply:
Now unit-tested.